package.json

```diff
@@ -102,8 +102,8 @@
     "@sentry/integrations": "^6.7.2",
     "@sentry/node": "^6.7.1",
     "@sentry/tracing": "^6.7.1",
-    "@stripe/react-stripe-js": "^1.10.0",
-    "@stripe/stripe-js": "^1.34.0",
+    "@stripe/react-stripe-js": "1.10.0",
+    "@stripe/stripe-js": "^4.1.0",
```

Comment:
💯 3 major versions bump!! I can't quickly tell from the release notes about breaking changes. Given it's a critical feature, I think it's worth QAing a bit on staging before releasing. I think Integrity covers the BNMO use cases pretty extensively, so we can probably manually test auction registration, card management in user settings, etc.